packages/ag-charts-community/src/chart/series/seriesLabelUtil.ts

```diff
@@ -0,0 +1,21 @@
+import { staticFromToMotion } from '../../motion/fromToMotion';
+import type { Node } from '../../scene/node';
+import type { Selection } from '../../scene/selection';
+import type { AnimationManager } from '../interaction/animationManager';
+
+type NodeWithOpacity = Node & { opacity: number };
+export function seriesLabelFadeInAnimation<T>(
+    { id }: { id: string },
+    animationManager: AnimationManager,
+    labelSelections: Selection<NodeWithOpacity, T>[]
+) {
+    const duration = animationManager.defaultDuration;
+    staticFromToMotion(
+        `${id}_empty-update-ready_labels`,
+        animationManager,
+        labelSelections,
+        { opacity: 0 },
+        { opacity: 1 },
+        { delay: duration, duration: animationManager.defaultDuration * 0.2 }
```

Comment:
Current requirements put this as hardcoded to 200ms, regardless the duration of the rest of the animation. Pending further expansion of https://ag-grid.atlassian.net/browse/AG-9189

Reply:
Ok - I'll switch back 👍

Reply:
Fixed - thanks for pointing that out